docs/usage.md

```diff
@@ -28,10 +28,10 @@ In order to cater to more specific needs, it is recommended to refer to the
 When initialising a client, to test, you can choose from 2 URLs:
 
 ### 1. **Testnet** (https://sdk-testnet.aepps.com)
-You can use this URL with any releasee on [npmjs](https://www.npmjs.com/package/@aeternity/aepp-sdk). It offers the last stable version of [Epoch](https://github.com/aeternity/epoch), used by all of of Aeternity's Dev Tools.
+You can use this URL with any releasee on [npmjs](https://www.npmjs.com/package/@aeternity/aepp-sdk). It offers the last stable version of [Node](https://github.com/aeternity/epoch), used by all of of Aeternity's Dev Tools.
```

Comment:
Same as above, the github url node is now https://github.com/aeternity/aeternity